### DISCUSSION TITLE
Mailing list

### DISCUSSION THREAD

**Discussion by Nuckerr** [NamelessMC/Nameless#2113] [Ideas]
NamelessMC has a nice mass emailing system however I would like to be able to send say newsletters to all my users but first I don't want to appear as spam to all the people who don't want it and secondly I believe its against the law to not have a unsubscribe button for those sorts of emails. (Don't quote me on that but literally every email has it). Being able to create multiple mail lists, send emails to everyone in it and being able to have an option to allow users to opt out of them (or opt in) in settings or something along that line would be very cool

**Comment by samerton** [comment]
Notification preferences added in 2.2.2, allowing for opt in/out of mass mailing!
